sources/engine/Stride.Particles/ParticlePool.cs

```diff
@@ -525,7 +526,6 @@ private unsafe void ReallocateForFieldRemoved(IntPtr oldPool, int oldCapacity, i
                     continue;
 
                 #error This is broken. This will AV.
```

Comment:
Someone knows why this is marked as broken? Seems to be equivalent to the old `CopyMemory` version, but as it is guarded by a `PARTICLES_SOA` constant that is not defined anywhere I don't know if it was a half-finished feature from Xenko days or it is somewhat functional.

Reply:
@ericwj Did you test this `PARTICLES_SOA` feature?

Reply:
@xen2 said in Discord that if that code is there since before the last modification, it doesn't matter that much. More so as it is disabled behind a feature flag. However, he doesn't know why the AV message @ericwj added.
Should I mark this as resolved? Maybe in the future we should revisit this "particles as structure-of-arrays" feature.

Reply:
Sure, let's resolve it